tangelo/toolboxes/molecular_computation/tests/test_molecule.py

```diff
@@ -198,6 +214,27 @@ def test_ecp(self):
         assert(molecule.n_active_electrons == 3)
         assert(molecule.n_active_mos == 35)
 
+    def test_mo_coeff_setter(self):
+        """Verify the dimension check in the mo_coeff setter."""
+
+        molecule = SecondQuantizedMolecule(H2_list, 0, 0, "sto-3g")
+
+        # Should work.
+        dummy_mo_coeff = np.array([
+            [0., 1.],
+            [2., 3.]
+        ])
+        molecule.mo_coeff = dummy_mo_coeff
+
+        # Should raise an AssertionError.
+        bad_dummy_mo_coeff = np.array([
```

Comment:
In efficient bumpy, this is `np.arange(9).reshape((3,3))`.
If we don't care about the numbers, why not just np.ones((3,3)) ? Stay simple.